react/tic-tac-toe/src/app/screens/Signup/index.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+
+import styles from './styles.module.scss';
+
+class Signup extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      lastName: '',
+      email: '',
+      password: '',
+      repeatPassword: ''
+    };
+    this.handleSignup = this.handleSignup.bind(this);
+    this.handleNameChange = this.handleNameChange.bind(this);
+    this.handleLastNameChange = this.handleLastNameChange.bind(this);
+    this.handleEmailChange = this.handleEmailChange.bind(this);
+    this.handlePasswordChange = this.handlePasswordChange.bind(this);
+    this.handleRepeatPasswordChange = this.handleRepeatPasswordChange.bind(this);
+  }
+
+  handleSignup(event) {
+    event.preventDefault();
+    console.log(event);
+    console.log(this.state);
+  }
+
+  handleNameChange(event) {
+    this.setState({ name: event.target.value });
+  }
+
+  handleLastNameChange(event) {
+    this.setState({ lastName: event.target.value });
+  }
+
+  handleEmailChange(event) {
+    this.setState({ email: event.target.value });
+  }
+
+  handlePasswordChange(event) {
+    this.setState({ password: event.target.value });
+  }
+
+  handleRepeatPasswordChange(event) {
+    this.setState({ repeatPassword: event.target.value });
+  }
+
+  render() {
+    return (
+      <div className={styles.signupContainer}>
+        <div className={styles.signup}>
+          <div className={styles.signupLogoContainer}>
+            <img className={styles.signupLogo} src="../../assets/logo-wolox.png" alt="logo" />
```

Comment:
You can import your images like this: 

`import Logo from '../../assets/logo-wolox.png';`

and then write the following: 

```suggestion
            <img className={styles.signupLogo} src={Logo} alt="logo" />
```